package.json

```diff
@@ -30,7 +30,7 @@
     "coverage": "node scripts/coverage.js",
     "install": "node scripts/install.js",
     "postinstall": "node scripts/build.js",
-    "pretest": "node_modules/.bin/jshint bin lib scripts test",
+    "pretest": "node_modules/.bin/eslint bin lib scripts test",
```

Comment:
FWIW, npm should add the "node_modules/.bin/" to the `$PATH` automatically, so you should only need `"eslint bin lib scripts test"` here.